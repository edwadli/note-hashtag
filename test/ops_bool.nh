if true || false then PrintEndline "Or :)" else PrintEndline "Or :("
be PrintEndline "And :)" unless true && false inwhichcase PrintEndline "And :("
if !false then PrintEndline "Not :)" else PrintEndline "Not :("
if !!true then PrintEndline "Double Not :)" else PrintEndline "Double Not :("
