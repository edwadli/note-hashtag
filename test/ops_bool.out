Or :)
And :)
Not :)
Double Not :)
